applications/DeepSpeed-Chat/training/utils/data/data_utils.py

```diff
@@ -84,7 +92,8 @@ def get_raw_dataset_split_index(local_rank, output_path, dataset_name, seed,
                                 split_name, data_split, split_index,
                                 data_size):
     index_file_name = f"{output_path}/{dataset_name}_seed{seed}_{split_name}_{data_split}_{split_index}.npy"
-    if not os.path.isfile(index_file_name):
+    # reindex each time since json file is more likely get modified
+    if (not os.path.isfile(index_file_name)) or (dataset_name == 'jsonfile'):
```

Comment:
I think this should actually be "(dataset_name == 'local/jsonfile')"? Please double check

Reply:
When ***get_raw_dataset_split_index*** is called in ***create_dataset*** in data_utils.py, the ***dataset_name*** passed into it is in fact ***raw_dataset.dataset_name_clean***.
 This ***dataset_name_clean*** doesn't contain a "/" in it as I find in other similar Dataset classes' defination. 
And I do run the code many times, it works here.